runtime/doc/vim_diff.txt

```diff
@@ -28,13 +28,14 @@ these differences.
 ==============================================================================
 2. Option defaults					  *nvim-option-defaults*
 
-- 'autoindent' is set by default.
+- 'autoindent' is set by default
 - 'autoread' is set by default
 - 'backspace' defaults to "indent,eol,start"
 - 'complete' doesn't include "i"
 - 'display' defaults to "lastline"
 - 'encoding' defaults to "utf-8"
 - 'formatoptions' defaults to "tcqj"
+- 'guioptions' doesn't include "t"
```

Comment:
This section is `Option defaults`, and while the default has changed, this should probably be moved down to `Removed features` for clarity. I'll open a PR.